public/javascripts/SVValidate/src/Main.js

```diff
@@ -177,7 +177,11 @@ function Main (param) {
         // There are certain features that will only make sense on desktop.
         if (!isMobile()) {
             svv.gsvOverlay = new GSVOverlay();
-            svv.keyboard = new Keyboard(svv.ui.validation);
+            if (svv.newValidateBeta) {
```

Comment:
You did this in one place but not in the other place where Keyboard is created, at the end of this file (when there are no labels left to validate).